druid/src/core.rs

```diff
@@ -513,18 +517,26 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                 };
 
                 if let Some(change) = this_changed {
-                    let event = LifeCycle::FocusChanged(change);
-                    self.inner.lifecycle(ctx, &event, data, env);
+                    // Only send FocusChanged in case there's actual change
+                    if old != new {
```

Comment:
if this is happening, maybe we should suppress it earlier, like at the window level?

Reply:
The problem is that we need to come down here to clear `state.request_focus`.